### ISSUE TITLE
When importing lib, IDE doesn't recognize it

### ISSUE THREAD

**Issue by MarcGodard** [feathersjs-ecosystem/dataloader#1]
When using `import { AppLoader } from 'feathers-dataloader'` the IDE doesn't recognize the library as installed even tho it is.

No errors, but I figured something is missing.

**Comment by marshallswain** [comment]
Yeah, there are some type declarations to work around in new TS projects.  We will eventually port feathers-dataloader into core, so we'll close this issue once that happens.
